daffodil-runtime1/src/main/scala/org/apache/daffodil/runtime1/api/Status.scala

```diff
@@ -0,0 +1,6 @@
+package org.apache.daffodil.runtime1.api
+
+object Status extends Enumeration {
+  type Status = Value
+  val DONE, READY, VISITING = Value
+}
```

Comment:
I'm wondering if we should just define all these enums that have to show up on JAPI, in Java, and use them from Scala as Java Enums.